app/helpers/application_helper.rb

```diff
@@ -2,6 +2,12 @@ module ApplicationHelper
   DEFAULT_PROGRAM_INCEPTION_DATE = Time.new(2018, 0o1, 0o1)
   STANDARD_DATE_DISPLAY_FORMAT = "%d-%^b-%Y"
 
+  def page_title
+    str = get_title_for_environment
+    str << " |" << content_for(:title) if content_for?(:title)
```

Comment:
I wonder if we can do `[ENV] <content> | Simple Dashboard`, or even remove `Simple Dashboard` from the titles? I'm looking at Github page titles for reference, and they don't mention "Github" in the page titles - I guess we let the icons do the talking.

Reply:
Yup, I'll add that for when there is an explicit title provided.